tutorial2_quantities_and_amounts/demo/main.js

```diff
@@ -0,0 +1,85 @@
+// data load
+d3.csv("../../data/surveyResults.csv", d => {
+  // clean up data - "+" converts the field to be a number
+  return {
+    timestamp: new Date(d.Timestamp),
+    python: +d["Python or R (or data analysis tool)"],
+    terminal: +d["Terminal (Bash/Zsh)"],
+    github: +d["Git / Github"],
+    html_css: +d["HTML / CSS"],
+    javascript: +d["Javascript"],
+    d3: +d["d3.js"],
+  };
+}).then(data => {
+  console.log(data);
+  /** CONSTANTS */
+  const width = window.innerWidth * 0.9,
+    height = window.innerHeight / 3,
+    paddingInner = 0.2,
+    margin = { top: 20, bottom: 20, left: 50, right: 50 },
+    duration = 1000,
+    delay = 20;
```

Comment:
this is good coding practice, but wondering if keeping them up here is disjointed from what is happening below. I think it makes sense for all css related items (`width`, `height`, `paddingInner`, `margin`) but maybe, just to keep it as simple as possible and this is a low hanging fruit, we could move `duration` and `delay` values to the code? This isn't a blocker but was just thinking -- trying to make the code below as obvious as possible (for example, someone who didn't notice we wrote this at the top might think `duration` is a JS defined item or a d3.js term)

Reply:
yea that's fair -- I was planning on having a slide in my deck talking about how we are going to start organizing our code and to specifically call out the idea of defining all the constants at the top. 

we can definitely pair it down though - whatever you think is best!